test/browser/browser.sh

```diff
@@ -75,6 +75,7 @@ SYSTEMD_SECCOMP=0 systemd-nspawn \
     -D /var/tmp/tasks/ \
     --ephemeral \
     --user user \
+    --setenv=TEST_AUDIT_NO_SELINUX="$TEST_AUDIT_NO_SELINUX" \
```

Comment:
Doesn't this need `${TEST_AUDIT_NO_SELINUX:-}` ? This script is `set -u`.

Reply:
Yes, let's do that.